website/app/components/doc/scroll-to-top/index.hbs

```diff
@@ -3,13 +3,13 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
-<button
+<a
+  href="#main"
   aria-label="Scroll back to top"
-  class="doc-scroll-to-top {{if this.isButtonVisible 'doc-scroll-to-top--is-visible'}}"
-  type="button"
-  tabindex={{if this.isButtonVisible 0 -1}}
+  class="doc-scroll-to-top {{if this.isLinkVisible 'doc-scroll-to-top--is-visible'}}"
```

Comment:
the fact that is a button or a link should be an implementation detail. what do you think of renaming the property to `isVisible` to avoid future refactorings if we change again our approach :)

Reply:
Good point! I'll update.